ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -157,6 +153,9 @@ cat > run_test_playbook.yaml <<-EOF
     ENVIRONMENT: "{{ lookup('env', 'ENVIRONMENT') }}"
     POST_INSTALL_COMMANDS: "{{ lookup('env', 'POST_INSTALL_COMMANDS') }}"
     ASSISTED_CONFIG: "{{ lookup('env', 'ASSISTED_CONFIG') }}"
+    ASSISTED_TEST_INFRA_IMAGE: "{{ lookup('env', 'ASSISTED_TEST_INFRA_IMAGE')}}"
+    ASSISTED_TEST_INFRA_SRC_PATH: "/go/src/github.com/openshift/assisted-test-infra"
```

Comment:
is it possible ditching this env?
for example, by copying from podman with relative path to where its at (because I do remember that's his working dir)

Reply:
:+1: I will check the workdir

Reply:
```
The podman cp command assumes container paths are relative to the container's root directory (/)
```
we need the complete path for `podman cp`

Reply:
hum... we could get the workdir from podman inspect and then build the path, I'll try

Reply:
Looks like it should do the trick :crossed_fingers: